src/controller/CommissioningDelegate.h

```diff
@@ -694,6 +694,9 @@ struct ReadCommissioningInfo
     bool isLIT                        = false;
     bool checkInProtocolSupport       = false;
     bool supportsConcurrentConnection = true;
+
+    BitMask<app::Clusters::IcdManagement::UserActiveModeTriggerBitmap> icdUserActiveModeTriggerHint;
+    CharSpan icdUserActiveModeTriggerInstruction;
```

Comment:
Follow-up: Better to put ICD metadata into one new struct, ICDManagementClusterInfo..

Reply:
https://github.com/project-chip/connectedhomeip/issues/31110

Reply:
I don't think putting it in a new struct in follow up is a good idea, will update the PR

Reply:
great, thanks